src/vs/workbench/services/editor/common/editorOverrideService.ts

```diff
@@ -91,15 +92,15 @@ export type RegisteredEditorOptions = {
 
 export type ContributedEditorInfo = {
 	id: string;
-	describes: (currentEditor: IEditorInput) => boolean;
+	describes: (override: string) => boolean;
```

Comment:
Instead of `override` should it be just something like "editorId"? The fact that this is an override is only evident as editor options.